workflow-service/src/test/java/com/redhat/parodos/workflow/version/WorkFlowVersionServiceImplTest.java

```diff
@@ -19,13 +17,17 @@ public void setUp() {
 	@Test
 	public void testGetHash() {
 		// given
+		// TODO passing objects to getHash method doesn't guarantee it's hash remains
+		// consistent, because java
+		// can change something in the implementation (or even compile flags) that will
+		// cause generation of a different
+		// digest, so we can't test reliably against a constant digest.
+		// If we want a stable test we probably need to change the argument of getHash
+		// from Object to some InputStream
 		Object object = new Object();
 
 		// when
-		assertDoesNotThrow(() -> {
```

Comment:
This is not the best approach to check the value, if we maintain the hardcoded hash for compatibility purposes because the workflows will be versioned in the database, we need to have a migration test.
If we go ahead with this change, the assertion test doesn't make sense and we should remove this test all together

Reply:
I left a comment to try to describe the problem. I agree this is not the best approach, however the test as it was has a flaw that will make it fail without a code change. 
The test checks the minimum it can so I don't think we should remove it, but instead fix it with implementation -  getHash on class doesn't guarantee a class change detection because simply changin a JVM even minor version would result a difference in the hash @RichardW98